mu-plugins/central-hub/src/config-store/internals.php

```diff
@@ -62,13 +62,22 @@ function _the_store( $store_key = '', $config_to_store = array(), $remove = null
  * Load a configuration from the filesystem, returning its
  * storage key and configuration parameters.
  *
- * @since 1.0.0
- *
  * @param string $path_to_file Absolute path to the config file.
  *
  * @return array
+ * @throws \Exception
+ * @since 1.1.0
+ *
  */
 function _load_config_from_filesystem( $path_to_file ) {
+	if ( ! $path_to_file ) {
```

Comment:
The Exception are thrown relates not to the path/to/file, but to what is returned from loading the file.  You want to check what is returned in `$config`.  In other words, you're checking what's inside of that configuration file.  More below....

Also, you don't need to check if `$path_to_file` is valid.  Why?  You are using a `require`.  [Look at the PHP manual](https://www.php.net/manual/en/function.require.php).  It says:

>`require` is identical to `include` except upon failure it will also produce a fatal `E_COMPILE_ERROR` level error.

If `$path_to_file` is empty or invalid, PHP will throw a fatal error when it tries to load.

Reply:
@hellofromtonya We _did_ talk several weeks back about the `require` control structure and how it would throw a fatal error if a value was not returned to the argument. That's why I thought the use of a conditional check on that argument would be appropriate. 

But you are right, it's not the argument `$path_to_file` that we're interested in. It's the contents of the configuration that needs checking.